src/components/header/styled.tsx

```diff
@@ -0,0 +1,61 @@
+import styled from 'styled-components';
+import { Box, Typography } from '@mui/material';
+import { colors, fonts } from '../../theme';
+
+const HeaderContainer = styled('header')`
```

Comment:
Not sure, need to double check, I think that in case if we use native html tag we need to use syntax like this styled.div

Reply:
If we import 'styled' from 'styled-components' than we need to do like styled.div. If we import 'styled' from MUI we can do everything in one syntax pattern -> styled()``.
fix: changed in all styled.ts files to one syntax (styled()``) - styled imported from MUI